src/test/java/com/bobocode/hoverla/bring/testsubject/beandefinition/classbased/ClassBasedBeanDefinitionNoName.java

```diff
@@ -0,0 +1,9 @@
+package com.bobocode.hoverla.bring.testsubject.beandefinition.classbased;
+
+import com.bobocode.hoverla.bring.annotation.Bean;
+import lombok.EqualsAndHashCode;
+
+@Bean
+@EqualsAndHashCode
```

Comment:
We don't expect users to implement equals and hashCode for their class based beans

Reply:
fixed